crates/sui/tests/ptb_files/simple/make_move_vec.ptb

```diff
@@ -0,0 +1,9 @@
+--gas-budget 1000
+--make-move-vec "<u64>" [1,2,3,4]
+--make-move-vec "<0x1::a::b>" [@0x1,@0x2]
+--make-move-vec "<address>" [@0x1,@0x2]
+--make-move-vec "<vector<address>>" [vector[@0x1],vector[@0x2]]
+--make-move-vec "<std::option::Option<u64>>" [none,none]
```

Comment:
Nice! Love that we properly resolve the named address here (and have the test to prove it!)